src/locales/en_US/en_US.json

```diff
@@ -1,4 +1,48 @@
 {
+    "workflow_complete": {
```

Comment:
Where are the files for all the other languages?

Reply:
The translation is going to happen for GA, however, saying for profile data the work is happening currently in a different branch that changes the labels based on country chosen. will follow after this branch is merged.